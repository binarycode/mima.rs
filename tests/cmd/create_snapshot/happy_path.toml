args = 'create-snapshot zero dev'
status.code = 1
fs.sandbox = true
