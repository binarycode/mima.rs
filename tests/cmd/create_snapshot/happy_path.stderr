error: failed to run 'ssh -o BatchMode=yes -o ConnectTimeout=10 -o ForwardAgent=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@10.210.0.13 qemu-img snapshot -cdev /mima/zero-sda.qcow2'

